src/main/scala/com/databricks/spark/sql/perf/mllib/classification/GBTClassification.scala

```diff
@@ -0,0 +1,48 @@
+package com.databricks.spark.sql.perf.mllib.classification
+
+import org.apache.spark.ml.{Estimator, ModelBuilder, Transformer}
+import org.apache.spark.ml.classification.GBTClassifier
+import org.apache.spark.ml.evaluation.{Evaluator, MulticlassClassificationEvaluator}
+
+import com.databricks.spark.sql.perf.mllib._
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+
+
+object GBTClassification extends BenchmarkAlgorithm
+  with TestFromTraining with TrainingSetFromTransformer with ScoringWithEvaluator {
+
+  def getFeatureArity(ctx: MLBenchContext): Array[Int] = {
```

Comment:
Will do.  I won't it make private anymore then so that I can use it for all trees and ensembles.